src/misc/crypt/crypt_ltc_mp_descriptor.c

```diff
@@ -19,9 +19,3 @@ ltc_math_descriptor ltc_mp = { 0 };
 // workaround for nasty osx linker bug
```

Comment:
I guess this comment should be removed too.

Reply:
Basically revert this commit: https://github.com/hunter-packages/libtomcrypt/pull/4/commits/e6af8a141f4818c06d6c1f70445f13cf6c5a51f3